jupyterfs/metamanager.py

```diff
@@ -40,22 +41,19 @@ class MetaManagerShared:
     def _files_handler_params_default(self):
         return {"path": self.root_dir}
 
-    def __init__(self, **kwargs):
+    def __init__(self, pyfs_kw=None, fsspec_kw=None, **kwargs):
         super().__init__(**kwargs)
         self._jupyterfsConfig = JupyterFsConfig(config=self.config)
 
         self._kwargs = kwargs
-        self._pyfs_kw = {}
 
         self.resources = []
         self._default_root_manager = self._jupyterfsConfig.root_manager_class(**self._kwargs)
         self._managers = dict((("", self._default_root_manager),))
 
         # copy kwargs to pyfs_kw, removing kwargs not relevant to pyfs
-        self._pyfs_kw.update(kwargs)
-        for k in (k for k in ("config", "log", "parent") if k in self._pyfs_kw):
-            self._pyfs_kw.pop(k)
```

Comment:
I think we need to retain this behavior to ensure traitlets kwargs gets passed correctly to the child-managers for all possible config routes.

Reply:
We can add it back in, we may need to separate pyfs_kw and fsspec_kw